src/whylogs/app/logger.py

```diff
@@ -290,7 +294,8 @@ def log(
             self,
             features: Optional[Dict[str, any]] = None,
             feature_name: str = None,
-            value: any = None
+            value: any = None,
+            constraints: DatasetConstraints = None,
```

Comment:
need to add docstring here

Reply:
eliminated constraints param - not needed